scripts/zkai_snarkjs_statement_envelope_benchmark.py

```
@@ -0,0 +1,591 @@
+    external_verify(proof, public_signals, verification_key)
+
+
+def verify_proof_only(
+    envelope: dict[str, Any],
+    external_verify: Callable[[dict[str, Any], list[Any], dict[str, Any]], None] = snarkjs_verify,
+) -> None:
+    proof, public_signals, verification_key = _snarkjs_payloads(envelope)
+    external_verify(proof, public_signals, verification_key)
+
+
+def classify_error(message: str) -> str:
```

Ruff (0.15.12)

[warning] Too many return statements (9 > 6)

(PLR0911)